plugins/org.eclipse.fordiac.ide.export.forte_ng.st/src/org/eclipse/fordiac/ide/export/forte_ng/st/StructuredTextSupport.xtend

```diff
@@ -297,7 +297,7 @@ abstract class StructuredTextSupport implements ILanguageSupport {
 
 	def protected dispatch CharSequence generateExpression(STBuiltinFeatureExpression expr) {
 		switch (expr.feature) {
-			case THIS: '''(*this)«IF expr.call»(«FOR arg : expr.generateCallArguments SEPARATOR ", "»«arg»«ENDFOR»)«ENDIF»'''
+			case THIS: '''this«IF expr.call»(«FOR arg : expr.generateCallArguments SEPARATOR ", "»«arg»«ENDFOR»)«ENDIF»'''
```

Comment:
why is the dereferencing not needed anymore?

Reply:
We are now already generating a `->` to access members of (internal) FBs: https://github.com/eclipse-4diac/4diac-ide/blob/24ac3a9a376b744d2b7239b2392e7e42afd0a170/plugins/org.eclipse.fordiac.ide.export.forte_ng.st/src/org/eclipse/fordiac/ide/export/forte_ng/st/StructuredTextSupport.xtend#L258

Reply:
But you are right about still needing it for the call. I will fix that.